Fix #125

BroBeansCPG opened the pull request:
**When merged this pull request will:**
fix #125

BroBeansCPG committed:
Fix #125

YannikH reviewed (changes requested):
This code also runs on the keybind for the landing and cockpit light being turned on, make sure this doesn't actually kill the landing or cockpit lights

BroBeansCPG commented:
Can you explain this a little more to me? Is this just to turn the landing light on or should it be doing more?
https://github.com/Project-Hatchet/public_h-60/blob/8fcf448558d1e27a15e6e557193aca0dca403d8d/addons/uh60_misc/functions/fnc_perFrame.sqf#L9-L17

And also, in an ideal world. 
1. When should the HMD indicator turn off? When both Pos & Landing is off? 
2. What should 'L' toggle? Just landing light?

BroBeansCPG committed:
New fix

"Lights" will only turn off if all lights (landing, pos & cockpit lights) are off

BroBeansCPG commented:
Should be good to go now
